Source/Scene/ModelExperimental/AlphaPipelineStage.js

```diff
@@ -23,16 +23,15 @@ AlphaPipelineStage.process = function (renderResources, primitive, frameState) {
 
   const renderStateOptions = renderResources.renderStateOptions;
   if (alphaOptions.pass === Pass.TRANSLUCENT) {
+    renderStateOptions.cull.enabled = false;
+    renderStateOptions.depthMask = false;
     renderStateOptions.blending = BlendingState.ALPHA_BLEND;
-  } else {
-    renderStateOptions.blending = BlendingState.DISABLED;
   }
```

Comment:
I removed the code that unsets the blending state because `BlendingState.DISABLED` is the default.